openpype/hosts/maya/api/lib.py

```diff
@@ -3379,3 +3380,45 @@ def dgcontext(mtime):
 def get_attribute_input(attr):
     connections = cmds.listConnections(attr, plugs=True, destination=False)
     return connections[0] if connections else None
+
+
+# Reference from Arnold
+# Get Image Information for colorspace
+def imageInfo(filepath):
+    """Take reference from makeTx.py
+    ImageInfo(filename): Get Image Information
+    AiTextureGetFormat(filename): Get Texture Format
+    AiTextureGetBitDepth(filename): Get Texture Bit Depth
+    """
+
+    # Get Texture Information
+    img_info = {}
+    img_info['filename'] = filepath
+    if os.path.isfile(filepath):
+        img_info['bit_depth'] = AiTextureGetBitDepth(filepath)     # noqa
+        img_info['format'] = AiTextureGetFormat(filepath)     # noqa
+    else:
+        img_info['bit_depth'] = 8
+        img_info['format'] = "unknown"
+    return img_info
+
+
+def guess_colorspace(img_info):
+    ''' Take reference from makeTx.py
+    Guess the colorspace of the input image filename.
+    @return: a string suitable for the --colorconvert
+            option of maketx (linear, sRGB, Rec709)
+    '''
```

Comment:
Instead of guessing, maybe we should expose some default settings so studios can override.